src/test/resources/features/MoiseiTextualStopper.feature

```diff
@@ -0,0 +1,43 @@
+@quiz
+Feature: Quiz Predefined
+  Background:
+    Given I open "login" page
+    When I type "rroux000@hulas.us" into email field
+    And I type "Password1" into password field
+    When I click button "Sign In"
+    And I wait for 1 sec
+    Then user role "TEACHER" is displayed
+    When I click "Quizzes" menu item
+    Then url contains text "quiz"
+    And I wait for 1 sec
+    When I click button "Create New Quiz"
+    And I wait for 1 sec
+
+  @quiz1
+  Scenario: Create a Textual Quiz with no Showstopper
+    When I type "Demo Texual Quiz Moisei" into title field
+    And I add a question
+    When I select "Textual" question in "Q1"
+    And I type "Question 1" into question field of "Q1"
+    And I click button "Save"
+    And I wait for 1 sec
+    Then quiz "Demo Texual Quiz Moisei" is displayed on the list of quizzes
+    And I delete "Demo Texual Quiz Moisei" from the list of quizzes
+
+
+  @quiz2
+    Scenario: Create a Textual Quiz with Showstopper
+    When I click "Quizzes" menu item
+    Then url contains text "quiz"
+    And I wait for 1 sec
+    When I click button "Create New Quiz"
+    And I wait for 1 sec
+    When I type "Showstopper Texual Quiz Moisei" into title field
+    And I add a question
+    When I select "Textual" question in "Q1"
+    And I type "Question 1" into question field of "Q1"
+    And I select Show-Stopper option
+    And I click button "Save"
+    And I wait for 1 sec
+    Then quiz "Showstopper Texual Quiz Moisei" is displayed on the list of quizzes
+    And I delete "Showstopper Texual Quiz Moisei" from the list of quizzes
```

Comment:
Please add verification of the red label in  Preview 
and one more scenario showing that only one showstopper per quiz is allowed.